### DISCUSSION TITLE
Split video track loses the frame contents at split point?

### DISCUSSION THREAD

**Discussion by eventualbuddha** [keiwando/toonsquid#613] [Q&A] [answered]
I imported a video to draw on top of and wanted to split it into scenes, so I started splitting the track up into sections at each of the transition points. However, I noticed that the frame that I split at seems to have its contents replaced by the one that comes after it. For example, if I split on frame 260 then looking at frame 260 after the split it contains what used to be in frame 261. The total number of frames does not appear to change, and no other frame contents appear to change. So frame 260 now looks exactly the same as 261. Perhaps there’s an off-by-1 error somewhere in the splitting code as relates to video files? Or maybe I’m just doing it wrong. Any ideas?

**Comment by keiwando** [comment, marked as answer]
Does the video have the same exact frame rate as the main timeline? When you split a drawing with a symbol layer (which is the layer in the main timeline that references the separate animation clip with the video in it), the symbol layer's [Time](https://toonsquid.com/handbook/symbols/symbol_layer/#time) property is automatically adjusted so that the new symbol layer in the second half of the split starts playing on the correct frame. However, since the time property only supports whole number increments, if the frame rate of the timeline and the video doesn't match exactly, this could cause a slightly different frame of the video to now play on that frame.

Please let me know in case the frame rate of the timeline and video are not different. So far I have not been able to reproduce the issue you describe in such a case.

**Reply by eventualbuddha** [reply]
It turns out they were different. The video I had imported had a frame rate of 23.973 fps, but most of the ToonSquid UI says 24 fps for that video. When I created an animation with 24 fps, I encountered the problem I described originally. When I made a new animation with the correct frame rate the problem went away.
